src/Quasar.jl

```diff
@@ -427,6 +434,19 @@ parse_oct_literal(token, qasm)     = QasmExpression(:integer_literal, tryparse(I
 parse_bin_literal(token, qasm)     = QasmExpression(:integer_literal, tryparse(Int, qasm[token[1]:token[1]+token[2]-1]))
 parse_float_literal(token, qasm)   = QasmExpression(:float_literal, tryparse(Float64, qasm[token[1]:token[1]+token[2]-1]))
 parse_boolean_literal(token, qasm)   = QasmExpression(:boolean_literal, tryparse(Bool, qasm[token[1]:token[1]+token[2]-1]))
+function parse_duration_literal(token, qasm)
+    str = String(codeunits(qasm)[token[1]:token[1]+token[2]-1])
+    duration = if endswith(str, "ns")
```

Comment:
do we need to handle `dt` here?

Reply:
Probably yes :)

Reply:
`dt` is a bit weird on a simulator because it's the "native time type" for the hardware... let me look what the spec says about what that should default to